src/main/java/carpet/script/api/Auxiliary.java

```diff
@@ -820,7 +823,15 @@ else if(lv.size()==2)
             Value retVal;
             if (fdesc.getMiddle().equals("nbt"))
             {
-                Tag state = ((CarpetScriptHost) c.host).readFileTag(fdesc.getLeft(), fdesc.getRight());
+                Tag state;
+                try
+                {
+                    state = ((CarpetScriptHost) c.host).readFileTag(fdesc.getLeft(), fdesc.getRight());
+                } catch (CrashException e)
```

Comment:
How did you get that exception to be thrown here? Ideally all exceptions in this call should have been caught and return null at the end.

Reply:
There are other IOExceptions here in there that should cause the exception to be rethrown, rather than retuning null.

Reply:
`NbtIo#read(DataInput, depth, PositionTracker)` directly throws the crash. It's not when the file doesn't exist or similar, but when it is corrupted, invalid or whatever. Should I catch it in `FileModule` instead? The problem with returning `null` is it cannot distinguish between invalid data and not found files.

Reply:
ok